style.css

```diff
@@ -450,3 +450,139 @@
   border: 1px solid #f55800;
   color: #f55800;
 }
+
+.contact-section {
+  padding: 80px 24px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact-paragraph {
+  font-family: "Inter", sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  padding: 0.1px 3px 40px 3px;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  gap: 30px;
+  align-items: center;
+  margin-top: 20px;
+}
+
+.form-input {
+  height: 48px;
+  width: 100%;
+  border: none;
+  border-bottom: 1px solid #dbd8d7;
+}
+
+#text-comment {
+  width: 100%;
+  height: 114px;
+  background: #fbf8f7;
+  border: none;
+}
+
+.form-input::placeholder {
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+  color: #979493;
+  padding-left: 16px;
+  padding-top: 15px;
+}
+
+#text-comment::placeholder {
+  font-style: italic;
+  color: #3c3a39;
+  padding-top: 12px;
+  padding-left: 12px;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+}
+
+.form-input:focus {
+  outline: none;
+  border-bottom: 1px solid #ff6b00;
+  padding-left: 16px;
+  padding-top: 15px;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+}
+
+#text-comment:focus {
+  padding-top: 12px;
+  padding-left: 12px;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+}
+
+.btn-get-in-touch {
+  align-items: center;
+  width: 131px;
+  height: 48px;
+  background: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  padding: 12px;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  color: #fff;
+  border: 1px solid #ff6b00;
+}
+
+.footer-class {
+  padding: 0.1px 24px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.footer-social-media {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  gap: 18px;
+  padding: 30px 0.1px;
+}
+
+#botom-line {
+  margin: 20px 0.1px 20px;
+  width: 134px;
+  height: 5px;
+  left: 120px;
+}
+
+#footer-separator {
+  color: #dfe1e6;
+}
+
+.btn-get-in-touch:hover {
+  background-color: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+.btn-get-in-touch:active {
+  background-color: #e05304;
+  color: #f4f1f0;
+}

```

Comment:
Nice job styling your project to match the given design👍. But there are still some improvements we can make to make our codebase look professional. Currently, I noticed there are a lot of repetitive styles and this is not a good CSS practice as you ought to keep your code DRY. this helps you save some lines of code and make your codebase easily maintainable. for instance, all buttons have the same font-color and border both in hover, active and normal state, Hence you can just use one class to represent that in your styles and call it in each of the buttons 😉. In your case, you are repeating the same style in different classes and this is not a good `CSS` practice😉.

<img width="384" alt="image" src="https://github.com/FilbertKimasa/Portfolio/assets/64643995/0c7dcb3d-26e0-4b0d-84e5-dac811dd09a9">

Kindly ensure your code is DRY, use one class for every similar style and call it in the respective HTML element😉.